pkg/console/dashboard_panels.go

```diff
@@ -330,6 +313,17 @@ func chartIsInstalled() bool {
 	return processed >= 1
 }
 
+func isAPIReady(managementURL, path string) bool {
+	if !strings.HasPrefix(current.managementURL, "https://") {
+		return false
+	}
+	command := fmt.Sprintf(`curl -fk %s%s`, managementURL, path)
+	cmd := exec.Command("/bin/sh", "-c", command)
```

Comment:
What about using the net/http package instead of the hardcode command?

Reply:
curl is simple and strong